vars/runATH.groovy

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env groovy
+
+/**
+ * Simple wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
```

Comment:
Recommend no default value. You should have to call out to reviewers that you are using a branch, so they can reject your PR until you fix that.

Reply:
We can discuss that, but I prefer to default to master, the reason is that I want to encourage developers to take ownership of relevant ath tests, by always using a tag or commit that you know works tests may break without you noticing. And IMO that is bad, very very bad and one of the causes of the current status of the ATH (as it happened with PCT)

My ideal setup would be to use a fixed commit id for a plugin PR lifecycle (as I fully understand determinism is very important) but to use master of ath when you are building your plugin master branch

Reply:
> tests may break without you noticing

That is why we have CI. To catch problems with…integrations. A **separate job** which builds lots of `master` branches together and looks for upcoming problems.

> use master of ath when you are building your plugin master branch

No no no, this will make builds unreproducible.